frontend/server/src/DAO/Identities.php

```diff
@@ -74,10 +77,14 @@ public static function findByUsernameOrName(string $usernameOrName): array {
             LIMIT 100";
         $args = [$usernameOrName, $usernameOrName, $usernameOrName, $usernameOrName];
 
+        /** @var list<array{country_id: null|string, current_identity_school_id: int|null, gender: null|string, identity_id: int, language_id: int|null, name: null|string, password: null|string, state_id: null|string, user_id: int|null, username: string}>|null $rs */
```

Comment:
```suggestion
        /** @var list<array{country_id: null|string, current_identity_school_id: int|null, gender: null|string, identity_id: int, language_id: int|null, name: null|string, password: null|string, state_id: null|string, user_id: int|null, username: string}> $rs */
```

`GetAll()` nunca regresa `null`.

Reply:
Listo